Fix:

warning: implicit conversion of NULL constant to 'bool' [-Wnull-conversion]

https://github.com/erincatto/box2d/issues/303
https://github.com/erincatto/box2d/commit/34ff60b55532882768ddeba11114caf2d2a529bb
--- src/libraries/Box2D/Collision/Shapes/b2ChainShape.h
+++ src/libraries/Box2D/Collision/Shapes/b2ChainShape.h
@@ -95,8 +95,8 @@ inline b2ChainShape::b2ChainShape()
 	m_radius = b2_polygonRadius;
 	m_vertices = NULL;
 	m_count = 0;
-	m_hasPrevVertex = NULL;
-	m_hasNextVertex = NULL;
+	m_hasPrevVertex = false;
+	m_hasNextVertex = false;
 }
 
 #endif
